python/railjson_generator/railjson_generator/schema/infra/speed_section.py

```diff
@@ -1,5 +1,5 @@
 from dataclasses import dataclass, field
-from typing import List
+from typing import List, Mapping
```

Comment:
Are these your change ?

Reply:
Yes, the changes here will be useful in future tests.